src/App.js

```diff
@@ -1,20 +1,27 @@
 import React from 'react';
 import { BrowserRouter, Route, Switch } from "react-router-dom";
+import { constants } from "./constants";
```

Comment:
Why?

Reply:
I go back and forth on this, usually I prefer defaults. I've also read opinions that exporting them with names helps maintain the upper case naming required for components. Changed to be defaults.